### ISSUE TITLE
About frames number of dataset

### ISSUE THREAD

**Issue by erwangccc** [Atze00/MoViNet-pytorch#17]
According to paper, "For all datasets, we train with 64 frames (except when the inference frames are fewer) at various frame-rates, and run inference with the same frame-rate". Frames of training dataset are 64. but frames of one video from my dataset may be less than 32, may be it's 8, 10,13, 17 etc. so is it ok?

Thanks in advance.

**Comment by Atze00** [comment]
It's generally ok to train on another dataset with different frames. You don't have to input the same number of frames of the original paper.

**Comment by erwangccc** [comment]
Thanks for your reply.

And there is another confusion. In the paper, "Similar to training, we use the stream buffer to cache activations between subclips. However, we can set the subclip length to a single frame (Tclip = 1) for maximum memory savings".  Is it means we can train with Tclip = 8 and inference with Tclip = 1, training frames number is different with inference frames number? 

If we want to inference with Tclip = 1, we must train with Tclip = 1?

**Comment by Atze00** [comment]
You can train with Tclips=8 and inference with Tclip = 1, whitout any change in accuracy.

**Comment by erwangccc** [comment]
> You can train with Tclips=8 and inference with Tclip = 1, whitout any change in accuracy.

thanks
